### PR TITLE
Feature/item page

### PR DESCRIPTION
**Pull request by madvingovich**

### CODE REVIEW

File: src/utils/apiStructure.js [hunk 1]
```diff
@@ -1,5 +1,22 @@
...
+  },
+  series: {
+    get: 'getSeriesById'
+  },
+  creators: {
+    get: 'getCreatorById'
+  },
+  all: {
+    search: 'getRandomCharacters'
+  }
```
Comment (dimkabuki) [inline src/utils/apiStructure.js:21]: Such a nice structure! 👍 
Very informative! 😄

Comment (madvingovich) [reply src/utils/apiStructure.js:21]: Is it a joke?)

Comment (madvingovich) [reply src/utils/apiStructure.js:21]: > Such a nice structure! 👍
> Very informative! 😄

Is it a joke?)

Comment (madvingovich) [reply src/utils/apiStructure.js:21]: Probably I should rename "get" to "getById"

Comment (dimkabuki) [reply src/utils/apiStructure.js:21]: 1. Nope, I was really glad to see a kind of `model` structure here. Most time this is a self documented peace of code which helps to understand how app is working and what it's doing
2. yep, `getById` would be more informative 👍

**Pull request merged**